keepassxc-browser/background/page.js

```diff
@@ -14,6 +14,7 @@ const defaultSettings = {
     credentialSorting: SORT_BY_GROUP_AND_TITLE,
     defaultGroup: '',
     defaultGroupAlwaysAsk: false,
+    downloadFaviconAfterSave: false,
```

Comment:
I didn't realize this was disabled by default and thought the code was broken. Perhaps we should enable this by default instead?

Reply:
On the other hand I would assume users don't want to make any network connections from KeePassXC by default.